search-index-import-util/src/main/java/no/unit/nva/search/ImportToSearchIndexHandler.java

```diff
@@ -0,0 +1,34 @@
+package no.unit.nva.search;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestStreamHandler;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import nva.commons.core.JsonUtils;
+
+public class ImportToSearchIndexHandler implements RequestStreamHandler {
```

Comment:
For the time being this lambda will run only when the index is corrupt and we need to re-index the whole table. No endpoint has been created for the time being. In the future we could create an endpoint that invokes asynchronously this function.